pallets/pallet-asset-switch/src/lib.rs

```diff
@@ -152,6 +184,9 @@ pub mod pallet {
 	where
 		LocalCurrencyBalanceOf<T, I>: Into<u128>,
 	{
+		/// Set a new switch pair.
+		///
+		/// See the crate's README for more.
```

Comment:
is it possible to link this somehow?

Reply:
Yeah I tried many different ways, but none of them was working, at least not from VS Code...

Reply:
Please check https://github.com/KILTprotocol/kilt-node/pull/667/commits/83e08dd6e0a0d116e74171cd1869061be31b8d21 for all your comments.

Reply:
@ChrisChinchilla can we resolve this comment as well?

Reply:
Not a very satisfactroy resolution, but sure.